### PR TITLE
Create foo.py

### PR DESCRIPTION
**Pull request by wanda-carlson**

### CODE REVIEW

**Comment by wanda-carlson** [comment]
G1

**Comment by wanda-carlson** [comment]
This image
782-536x354

**Comment by wanda-carlson** [comment]
S1 **Henry Richard**

File: src/foo.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Foo:
+  def hello():
+    print("Hello foo")
```
Comment (henry-richard) [inline src/foo.py:3]: RC1

**Review by henry-richard — CHANGES_REQUESTED** [review]
R1

**Comment by henry-richard** [comment]
S3

File: src/foo.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class Foo:
+  def hello():
```
Comment (henry-richard) [inline src/foo.py:2]: RC3

**Review by henry-richard — CHANGES_REQUESTED** [review]
R3

**Review by henry-richard — APPROVED** [review]
lgtm

**Comment by henry-richard** [comment]
**bold **_italics__** **_~strike~

**Comment by henry-richard** [comment]
**bold**    _italics_

**Comment by henry-richard** [comment]
**bold** _italics_

**Comment by henry-richard** [comment]
**bold** _italics_ ~strike~

**Comment by henry-richard** [comment]
>Indent

**Comment by henry-richard** [comment]
```
func code() {
```

**Comment by henry-richard** [comment]
Image
[782-536x354.jpeg](https://files.slack.com/files-pri/T0289BY1FK4-F02T121J06A/782-536x354.jpeg)

**Comment by wanda-carlson** [comment]
- [ ] This
- [ ] That
- [ ] Other

**Comment by wanda-carlson** [comment]
1. a

**Comment by wanda-carlson** [comment]
b

**Comment by wanda-carlson** [comment]
1. a
2. b
3. c

**Comment by henry-richard** [comment]
* - [ ] hello

**Comment by henry-richard** [comment]
- [ ] hi

**Comment by henry-richard** [comment]
- [ ] todo
- [x] done

**Comment by wanda-carlson** [comment]
this is `good`

**Comment by henry-richard** [comment]
this is `better`

**Pull request merged**